templates/jaasai/community.html

```diff
@@ -74,8 +74,8 @@ <h3 class="p-media-object__title">
             <p class="p-media-object__content">Affiliation: AppScale</p>
           </div>
         </div>
-        <p>Chris Donati has been working on improving the <a href="{{ url_for('jaasstore.details', charm_or_bundle_name='appscale', series_or_version='precise', version=1) }}">AppScale platform</a>, which lets you deploy your Google App Engine application on any cloud, public or private. He lives in Santa Barbara, California, where he&rsquo;s been trying to get better at the game of Go.</p>
-        <a href="{{ url_for('jaasstore.details', charm_or_bundle_name='appscale', series_or_version='precise', version=1) }}" class="p-button--positive is-external">Contribute</a>
+        <p>Chris Donati has been working on improving the AppScale platform, which lets you deploy your Google App Engine application on any cloud, public or private. He lives in Santa Barbara, California, where he&rsquo;s been trying to get better at the game of Go.</p>
+        <a href="https://github.com/AppScale/gts" class="p-button--positive is-external">Contribute</a>
```

Comment:
👍  for removing redundant `a` tags, will result in a better experience for screen reader users when navigating by links.

As a side note, lots of links go to external domains, should we be adding the `.p-link--external` class?

> The .p-link--external class should be used on hyperlinks that go to a different domain than the current one.
https://vanillaframework.io/docs/patterns/links#external

Reply:
Yeah I wondered if I should but with the deprecation of the icon in the next version of Vanilla. It might not be worth it?

Reply:
Right, didn't know this icon will be deprecated. Porbably doesn't make sense in this case.